cli/src/main/scala/com/advancedtelematic/tuf/cli/Cli.scala

```diff
@@ -77,6 +78,10 @@ object Cli extends App with VersionInfo {
     }
     .text("required for all subcommands")
 
+    opt[Unit]("verbose").action { (_, c) =>
```

Comment:
I think the verbose config should configure the logger level and then everything we don't need to check every time we log, just log to debug if it should be logged on verbose only.

Reply:
Updated.